egs2/TEMPLATE/spk1/spk.sh

```diff
@@ -59,6 +60,10 @@ spk_tag=              # Suffix to the result dir for spk model training.
 spk_config=           # Config for the spk model training.
 spk_args=             # Arguments for spk model training.
 
+# Inference related
+inference_config=     # Inference configuration file
+inference_model_file= # Inference model weight file
```

Comment:
Better to follow the convention in other tasks to use the name `inference_model` and set the default value for it (e.g., `valid.eer.best.pth`).